.github/workflows/testAPI.yml

```
@@ -0,0 +1,62 @@
+on:
+  pull_request:
+    paths:
+      - "API/**"
+  workflow_dispatch:
+
+jobs:
+  unit-testing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)